Bluepill-runner/Bluepill-runner/BPRunner.m

```diff
@@ -267,7 +286,7 @@ - (int)runWithBPXCTestFiles:(NSArray<BPXCTestFile *> *)xcTestFiles {
                 [taskList addObject:[NSString stringWithFormat:@"%lu", taskNumber]];
                 [self.nsTaskList addObject:task];
                 [bundles removeObjectAtIndex:0];
-                [BPUtils printInfo:INFO withString:@"Started Simulator %lu (PID %d).", taskNumber, [task processIdentifier]];
+                [BPUtils printInfo:INFO withString:@"Started BP-%lu (PID %d).", taskNumber, [task processIdentifier]];
```

Comment:
Is it possible to print any info about the test that's starting? It'd be super useful to debug things like #323

Reply:
Do you mean which test suite?

Reply:
👍

Reply:
Ok, done. Now it prints:
```
{37345} 20190411.163820 [  INFO  ] (BP-2) Starting TestSuite PeopleTimeUtilsTests
```